cmd/performance-profile-creator/cmd/root.go

```diff
@@ -131,10 +152,23 @@ func getProfileData(args *profileCreatorArgs) (*ProfileData, error) {
 		isolatedCPUs:           isolatedCPUs,
 		nodeSelector:           mcp.Spec.NodeSelector,
 		performanceProfileName: args.profileName,
+		topologyPoilcy:         args.tmPolicy,
 	}
 	return profileData, nil
 }
 
+func validateFlag(value string, validValues []string) error {
+	for _, validValue := range validValues {
+		if strings.ToLower(value) == validValue {
```

Comment:
`strings.EqualFold` (https://golang.org/pkg/strings/#EqualFold) will probably come handy here

Reply:
the string matching should be extracted:

```
func validateFlag(value string, validValues []string) error {
	if isStringInSlice(value, validValues) {
		return nil
       }
	return fmt.Errorf("Value '%s' is invalid for flag 'color'. Valid values "+
		"come from the set %v", value, validValues)
}

func isStringInSlice(value string, candidates []string) bool {
	for _, candidate := range candidates {
		if strings.EqualFold(candidate, value) {
			return true
		}
	}
	return false
}
```

Reply:
Thanks, will fix